skynet/bandwidth_test.go

```diff
@@ -24,18 +33,40 @@ func TestNumChunks(t *testing.T) {
 	}
 }
 
+// TestRawStorageUsed ensures that RawStorageUsed works as expected.
+func TestRawStorageUsed(t *testing.T) {
+	tests := []struct {
+		size   int64
+		result int64
+	}{
+		{size: 0, result: baseSectorTotalSize},
+		{size: 1 * MiB, result: baseSectorTotalSize},
+		{size: 4 * MiB, result: baseSectorTotalSize},
+		{size: 5 * MiB, result: baseSectorTotalSize + chunkTotalSize},
+		{size: 50 * MiB, result: baseSectorTotalSize + 2*chunkTotalSize},
+		{size: 500 * MiB, result: baseSectorTotalSize + 13*chunkTotalSize},
+	}
+	for _, tt := range tests {
+		res := RawStorageUsed(tt.size)
+		if res != tt.result {
+			t.Errorf("Expected a %d MiB file to result into %d MiB used for upload storage, got %d MiB.",
+				tt.size/MiB, tt.result/MiB, res/MiB)
+		}
+	}
+}
+
 // TestStorageUsed ensures that StorageUsed works as expected.
 func TestStorageUsed(t *testing.T) {
 	tests := []struct {
 		size   int64
 		result int64
 	}{
-		{size: 0, result: SizeBaseSector},
-		{size: 1 * MiB, result: SizeBaseSector},
-		{size: 4 * MiB, result: SizeBaseSector},
-		{size: 5 * MiB, result: SizeBaseSector + SizeChunk},
-		{size: 50 * MiB, result: SizeBaseSector + 2*SizeChunk},
-		{size: 500 * MiB, result: SizeBaseSector + 13*SizeChunk},
+		{size: 0, result: baseSectorTotalSize / RedundancyAPIDivisor},
+		{size: 1 * MiB, result: baseSectorTotalSize / RedundancyAPIDivisor},
+		{size: 4 * MiB, result: baseSectorTotalSize / RedundancyAPIDivisor},
+		{size: 5 * MiB, result: (baseSectorTotalSize + chunkTotalSize) / RedundancyAPIDivisor},
```

Comment:
Is it correct that we divide by `RedundancyAPIDivisor` (3) for both redundancies (3 and 10)?

Reply:
Yes, that was an agreed compromise. We'll be moving to a more transparent (and complicated) representation soon.